bt/cmd/rds_gui

```diff
@@ -50,11 +50,7 @@ default_arg() { if [ -z "$OPTARG" ]; then
     die "No arg for --$OPT option" && usage; fi; 
 }
 
-# lib/utils
-export BT=${HOME}/.bt
-. "${BT}/lib/utils.bash"
-
-while getopts P:d:f:g:h:j:k:o:p:t:u:v:-: OPT; do
+while getopts :P:f:g:j:k:o:p:t:u:v:-:hd OPT; do
```

Comment:
lines 54 and 55 are repeats of lines 16 and 17.  Fixed syntax errors to enable OPTs processing

Reply:
beautiful !   Thanks for noticing this rather subtle, but pernicious flaw.